tools/db_stress.cc

```diff
@@ -198,6 +205,10 @@ DEFINE_int32(max_background_compactions,
              "The maximum number of concurrent background compactions "
              "that can occur in parallel.");
 
+DEFINE_int32(num_bottom_pri_threads, 0,
```

Comment:
Shouldn't the default be 1 here? as per: https://github.com/sagar0/rocksdb/blob/master/include/rocksdb/env.h#L365-L366